projects/1.19/assets/crimsonsteves-more-mobs/crimson_steves_mobs/lang/zh_cn.json

```
@@ -0,0 +1,133 @@
+{
+  "entity.crimson_steves_mobs.animated_temple": "§8复苏的神庙",
+  "entity.crimson_steves_mobs.big_snow_golem": "大型雪傀儡",
+  "entity.crimson_steves_mobs.chad_creeper_golem": "苦力怕傀儡斗士",
+  "entity.crimson_steves_mobs.chad_villager": "村民斗士",
+  "entity.crimson_steves_mobs.creeckin": "苦力鸡",
+  "entity.crimson_steves_mobs.creeper_golem": "苦力怕傀儡",
+  "entity.crimson_steves_mobs.creepig": "苦力猪",
+  "entity.crimson_steves_mobs.creesteve": "苦力史蒂夫",
+  "entity.crimson_steves_mobs.crude_redstone_golem": "粗糙的红石傀儡",
+  "entity.crimson_steves_mobs.crude_redstone_monstrosity": "§4粗糙的红石巨兽",
+  "entity.crimson_steves_mobs.cyborg_vindicator": "机械报复者",
+  "entity.crimson_steves_mobs.derpig": "Derpig",
+  "entity.crimson_steves_mobs.elite_ender_ravager": "精英末地掠夺兽",
+  "entity.crimson_steves_mobs.ender_ravager": "末地掠夺兽",
+  "entity.crimson_steves_mobs.explosive_phantom": "自爆幻翼",
+  "entity.crimson_steves_mobs.flying_chicken": "飞翔的鸡",
+  "entity.crimson_steves_mobs.giant_illager": "巨型灾民",
+  "entity.crimson_steves_mobs.lava_mutant_zombie": "熔岩突变僵尸",
+  "entity.crimson_steves_mobs.minion_redstone_golem": "红石奇兵",
+  "entity.crimson_steves_mobs.pavager": "帕瓦格",
+  "entity.crimson_steves_mobs.phantom_tamer": "幻翼驯兽师",
+  "entity.crimson_steves_mobs.pig_brute": "野蛮猪",
+  "entity.crimson_steves_mobs.ravager_enderman": "末影人掠夺兽",
+  "entity.crimson_steves_mobs.redstone_monstrosity": "§c红石巨兽",
+  "entity.crimson_steves_mobs.redstone_powered_iron_golem": "红石动力铁傀儡",
+  "entity.crimson_steves_mobs.t_rabus": "§7T-拉布斯",
```

```suggestion
  "entity.crimson_steves_mobs.t_rabus": "T-拉布斯",
```
和这个有相似之处吗
https://www.planetminecraft.com/mob-skin/t-rabus-boss/

---

一样的